From ddb80e50eca93e559ccae34ba7d72268a6cf5f00 Mon Sep 17 00:00:00 2001
From: Thomas Arcila <134677+tarcila@users.noreply.github.com>
Date: Thu, 16 Apr 2026 18:53:37 -0400
Subject: [PATCH 2/5] Find X11 and OpenGL as dependencies of OpenUSD/MaterialX

---
 UsdBridge/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/UsdBridge/CMakeLists.txt b/UsdBridge/CMakeLists.txt
index 556acd0..174a194 100644
--- a/UsdBridge/CMakeLists.txt
+++ b/UsdBridge/CMakeLists.txt
@@ -118,6 +118,8 @@ endif()
 set(PXR_FIND_OPENSUBDIV_IN_CONFIG OFF)
 
 # Search in config mode (USD will now skip TBB finding and use the targets defined above)
+find_package(X11)
+find_package(OpenGL)
 list(APPEND USD_INIT_TARGETS usdGeom usdVol usdShade usdLux usdUtils plug usdRender)
 if(${USD_DEVICE_RENDERING_ENABLED})
   find_package(OpenGL REQUIRED)
-- 
2.54.0

