From 8bf53354e635392c7690b6b0e5bbe1e17452b37d Mon Sep 17 00:00:00 2001
From: Thomas Arcila <134677+tarcila@users.noreply.github.com>
Date: Sun, 20 Jul 2025 18:19:17 -0400
Subject: [PATCH 3/6] Also skip gomp when flattening usd libraries

---
 UsdBridge/usdhelpers.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/UsdBridge/usdhelpers.cmake b/UsdBridge/usdhelpers.cmake
index 976aa1f..0d70b75 100644
--- a/UsdBridge/usdhelpers.cmake
+++ b/UsdBridge/usdhelpers.cmake
@@ -42,6 +42,7 @@ function(flatten_and_cleanup_targets RESULT_TARGETS INIT_TARGETS)
       AND NOT (${cur_target} MATCHES ${MATCH_SUFFIX})
       AND NOT (${cur_target} STREQUAL "dl")
       AND NOT (${cur_target} STREQUAL "m")
+      AND NOT (${cur_target} STREQUAL "gomp")
       )
       list(APPEND flat_targets ${cur_target})
       
-- 
2.49.0

