diff --git a/devices/rtx/CMakeLists.txt b/devices/rtx/CMakeLists.txt
--- a/devices/rtx/CMakeLists.txt
+++ b/devices/rtx/CMakeLists.txt
@@ -80,8 +80,8 @@
   message(FATAL_ERROR "Invalid/unknown version of OptiX selected")
 endif()
 
-anari_sdk_fetch_project(NAME optix_headers URL ${OPTIX_URL})
-appendSearchPaths(${optix_headers_LOCATION})
+#anari_sdk_fetch_project(NAME optix_headers URL ${OPTIX_URL})
+#appendSearchPaths(${optix_headers_LOCATION})
 
 ## Validate Neural Graphics Primitives support ##
 
