diff --git a/devices/rtx/CMakeLists.txt b/devices/rtx/CMakeLists.txt
index 95ff375..5acd21d 100644
--- a/devices/rtx/CMakeLists.txt
+++ b/devices/rtx/CMakeLists.txt
@@ -86,11 +86,11 @@ elseif(${OPTIX_FETCH_VERSION} STREQUAL "8.1")
 elseif(${OPTIX_FETCH_VERSION} STREQUAL "9.0")
   set(OPTIX_URL https://github.com/NVIDIA/optix-dev/archive/refs/tags/v9.0.0.zip)
 else()
-  message(FATAL_ERROR "Invalid/unknown version of OptiX selected")
+#  message(FATAL_ERROR "Invalid/unknown version of OptiX selected")
 endif()
 
-anari_sdk_fetch_project(NAME optix_headers URL ${OPTIX_URL})
-appendSearchPaths(${optix_headers_LOCATION})
+#anari_sdk_fetch_project(NAME optix_headers URL ${OPTIX_URL})
+#appendSearchPaths(${optix_headers_LOCATION})
 
 find_package(OptiX REQUIRED)
 
