From fc465102b1ed10019b23d28c57136dade84e3735 Mon Sep 17 00:00:00 2001
From: Thomas Arcila <134677+tarcila@users.noreply.github.com>
Date: Wed, 30 Apr 2025 21:06:35 -0400
Subject: [PATCH] Link with openvdb when needed

---
 cycles/src/device/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cycles/src/device/CMakeLists.txt b/cycles/src/device/CMakeLists.txt
index 431ae7a80..c220f28cb 100644
--- a/cycles/src/device/CMakeLists.txt
+++ b/cycles/src/device/CMakeLists.txt
@@ -233,6 +233,12 @@ endif()
 endif()

+if(WITH_CYCLES_NANOVDB)
+  list(APPEND LIB
+    openvdb
+  )
+endif()
+
 if(WITH_CYCLES_OSL)
   list(APPEND LIB
     ${OSL_LIBRARIES}
   )
-- 
2.49.0
