diff --git a/OSPRayDevice.cpp b/OSPRayDevice.cpp
index f96945f..f443c18 100644
--- a/OSPRayDevice.cpp
+++ b/OSPRayDevice.cpp
@@ -267,7 +267,7 @@ int OSPRayDevice::getProperty(ANARIObject object,
       m_state->commitBuffer.flush();
     }
     return helium::referenceFromHandle(object).getProperty(
-        name, type, mem, mask);
+        name, type, mem, size, mask);
   }
 
   return 0;
diff --git a/Object.cpp b/Object.cpp
index 14faf65..460a446 100644
--- a/Object.cpp
+++ b/Object.cpp
@@ -28,7 +28,7 @@ void Object::finalize()
 }
 
 bool Object::getProperty(
-    const std::string_view &name, ANARIDataType type, void *ptr, uint32_t flags)
+    const std::string_view &name, ANARIDataType type, void *ptr, uint64_t size, uint32_t flags)
 {
   if (name == "valid" && type == ANARI_BOOL) {
     helium::writeToVoidP(ptr, isValid());
diff --git a/Object.h b/Object.h
index 136b856..8ab19ab 100644
--- a/Object.h
+++ b/Object.h
@@ -21,6 +21,7 @@ struct Object : public helium::BaseObject
   bool getProperty(const std::string_view &name,
       ANARIDataType type,
       void *ptr,
+      uint64_t size,
       uint32_t flags) override;
 
   void commitParameters() override;
diff --git a/array/Array.cpp b/array/Array.cpp
index 0a106bc..0071091 100644
--- a/array/Array.cpp
+++ b/array/Array.cpp
@@ -138,7 +138,7 @@ void Array::markDataModified()
 }
 
 bool Array::getProperty(
-    const std::string_view &name, ANARIDataType type, void *ptr, uint32_t flags)
+    const std::string_view &name, ANARIDataType type, void *ptr, uint64_t size, uint32_t flags)
 {
   return 0;
 }
diff --git a/array/Array.h b/array/Array.h
index 1a89e81..651f631 100644
--- a/array/Array.h
+++ b/array/Array.h
@@ -57,6 +57,7 @@ struct Array : public helium::BaseArray
   virtual bool getProperty(const std::string_view &name,
       ANARIDataType type,
       void *ptr,
+      uint64_t size,
       uint32_t flags) override;
   virtual void commitParameters() override;
   virtual void finalize() override;
diff --git a/frame/Frame.cpp b/frame/Frame.cpp
index 16dee3b..707cbae 100644
--- a/frame/Frame.cpp
+++ b/frame/Frame.cpp
@@ -96,7 +96,7 @@ void Frame::initFB(const bool denoising)
 }
 
 bool Frame::getProperty(
-    const std::string_view &name, ANARIDataType type, void *ptr, uint32_t flags)
+    const std::string_view &name, ANARIDataType type, void *ptr, uint64_t size, uint32_t flags)
 {
   if (type == ANARI_FLOAT32 && name == "duration") {
     if (flags & ANARI_WAIT)
diff --git a/frame/Frame.h b/frame/Frame.h
index 2911fbd..0680de9 100644
--- a/frame/Frame.h
+++ b/frame/Frame.h
@@ -26,6 +26,7 @@ struct Frame : public helium::BaseFrame
   bool getProperty(const std::string_view &name,
       ANARIDataType type,
       void *ptr,
+      uint64_t size,
       uint32_t flags) override;
 
   void commitParameters() override;
diff --git a/scene/Group.cpp b/scene/Group.cpp
index bac32dc..40fe7db 100644
--- a/scene/Group.cpp
+++ b/scene/Group.cpp
@@ -24,7 +24,7 @@ Group::~Group()
 }
 
 bool Group::getProperty(
-    const std::string_view &name, ANARIDataType type, void *ptr, uint32_t flags)
+    const std::string_view &name, ANARIDataType type, void *ptr, uint64_t size, uint32_t flags)
 {
   if (name == "bounds" && type == ANARI_FLOAT32_BOX3) {
     if (flags & ANARI_WAIT) {
@@ -38,7 +38,7 @@ bool Group::getProperty(
     return true;
   }
 
-  return Object::getProperty(name, type, ptr, flags);
+  return Object::getProperty(name, type, ptr, size, flags);
 }
 
 void Group::commitParameters()
diff --git a/scene/Group.h b/scene/Group.h
index afaadde..1830457 100644
--- a/scene/Group.h
+++ b/scene/Group.h
@@ -18,6 +18,7 @@ struct Group : public Object
   bool getProperty(const std::string_view &name,
       ANARIDataType type,
       void *ptr,
+      uint64_t size,
       uint32_t flags) override;
 
   void commitParameters() override;
diff --git a/scene/World.cpp b/scene/World.cpp
index 1500c64..3ef6bf0 100644
--- a/scene/World.cpp
+++ b/scene/World.cpp
@@ -36,7 +36,7 @@ World::~World()
 }
 
 bool World::getProperty(
-    const std::string_view &name, ANARIDataType type, void *ptr, uint32_t flags)
+    const std::string_view &name, ANARIDataType type, void *ptr, uint64_t size, uint32_t flags)
 {
   if (name == "bounds" && type == ANARI_FLOAT32_BOX3) {
     if (flags & ANARI_WAIT) {
@@ -49,7 +49,7 @@ bool World::getProperty(
     return true;
   }
 
-  return Object::getProperty(name, type, ptr, flags);
+  return Object::getProperty(name, type, ptr, size, flags);
 }
 
 void World::commitParameters()
diff --git a/scene/World.h b/scene/World.h
index 2a59647..929647a 100644
--- a/scene/World.h
+++ b/scene/World.h
@@ -15,6 +15,7 @@ struct World : public Object
   bool getProperty(const std::string_view &name,
       ANARIDataType type,
       void *ptr,
+      uint64_t size,
       uint32_t flags) override;
 
   void commitParameters() override;
